rtic-sync/src/channel.rs

```diff
@@ -77,8 +77,61 @@ impl<T, const N: usize> Channel<T, N> {
         }
     }
 
+    /// Clear any remaining items from this `Channel`.
+    pub fn clear(&mut self) {
+        for _ in self.queued_items() {}
```

Comment:
To help readability, lets do `drop(item)` in the loop.

Reply:
Done